// The Firmament project
// Copyright (c) 2011-2012 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//
// Task spawn message.

syntax = "proto3";

package firmament;

import "base/task_desc.proto";

message TaskSpawnMessage {
  uint64 creating_task_id = 1;
  TaskDescriptor spawned_task_desc = 2;
}
